{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"rejected"},{"kind":"text","text":"("},{"kind":"externalParam","text":"cause"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":")"}],"languages":["swift"],"platforms":["iOS"]}]},{"kind":"parameters","parameters":[{"name":"cause","content":[{"type":"paragraph","inlineContent":[{"type":"text","text":"A string describing the reason for the rejection."}]}]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/blinqsdk\/traceerror\/rejected(cause:)"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/BlinqSDK\/documentation\/BlinqSDK\/TraceError\/rejected(cause:)","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"Represents a rejection of the trace by the server."}],"kind":"symbol","metadata":{"fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"rejected"},{"kind":"text","text":"("},{"kind":"externalParam","text":"cause"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":")"}],"title":"TraceError.rejected(cause:)","roleHeading":"Case","role":"symbol","symbolKind":"case","externalID":"s:8BlinqSDK10TraceErrorO8rejectedyACSS_tcACmF","modules":[{"name":"BlinqSDK"}]},"hierarchy":{"paths":[["doc:\/\/BlinqSDK\/documentation\/BlinqSDK","doc:\/\/BlinqSDK\/documentation\/BlinqSDK\/TraceError"]]},"references":{"doc://BlinqSDK/documentation/BlinqSDK":{"role":"collection","title":"BlinqSDK","abstract":[],"identifier":"doc:\/\/BlinqSDK\/documentation\/BlinqSDK","kind":"symbol","type":"topic","url":"\/documentation\/blinqsdk"},"doc://BlinqSDK/documentation/BlinqSDK/TraceError":{"role":"symbol","title":"TraceError","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"TraceError"}],"abstract":[{"type":"codeVoice","code":"TraceError"},{"type":"text","text":" defines a set of errors that can occur while handling traces."}],"identifier":"doc:\/\/BlinqSDK\/documentation\/BlinqSDK\/TraceError","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"TraceError"}],"url":"\/documentation\/blinqsdk\/traceerror"},"doc://BlinqSDK/documentation/BlinqSDK/TraceError/rejected(cause:)":{"role":"symbol","title":"TraceError.rejected(cause:)","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"rejected"},{"kind":"text","text":"("},{"kind":"externalParam","text":"cause"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":")"}],"abstract":[{"type":"text","text":"Represents a rejection of the trace by the server."}],"identifier":"doc:\/\/BlinqSDK\/documentation\/BlinqSDK\/TraceError\/rejected(cause:)","kind":"symbol","type":"topic","url":"\/documentation\/blinqsdk\/traceerror\/rejected(cause:)"}}}